src/js/hc-scripts/expire-invitations.js

```diff
@@ -0,0 +1,48 @@
+const { createHandler } = require('./utils');
+const { getInvitationExpiryCandidates, expireInvitation } = require('js/server/attendance/logic');
+
+function createExpiryQueue(responsesToProcess, dryRun) {
+  // Defensive clone for mutating array
+  const responses = responsesToProcess.slice(0);
+
+  const expireResponse = (response) => {
+    if (dryRun) {
```

Comment:
Yargs gives us help:

```
❯ npm run hc-script -- --help

> hack-cambridge-website@0.0.0 hc-script hack-cambridge-website
> export NODE_PATH=$NODE_PATH:./src; node ./hc-script || true "help"

Commands:
  create-admin                              Create a new admin user
  create-token <email>                      Create an oauth token for an admin
  suggest-responses <type> <limit>          Suggest a set of people to respond
  <outputfile>                              to and saves them to a file
  respond <type> <inputfile>                Takes a set of applications from a
                                            file and responds to them
  expire-invitations [--dry-run]            Take any pending invitations that
                                            are too old and expire them

Options:
  --help  Show help                                                    [boolean]
```

I'll add a description to that option in particular, though.